client/extensions/woocommerce/woocommerce-services/views/packages/add-package.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { concat, difference, flatten, map, omit, trim } from 'lodash';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
+import Dialog from 'components/dialog';
+import AddPackagePresets from './add-package-presets';
+import checkInputs from './modal-errors';
+import inputFilters from './input-filters';
+import FieldError from '../../components/field-error';
+
+const getDialogButtons = ( mode, dismissModal, savePackage, onRemove, translate ) => {
+	const buttons = [
+		<FormButton onClick={ savePackage }>
+			{ ( 'add' === mode ) ? translate( 'Add package' ) : translate( 'Apply changes' ) }
+		</FormButton>,
+		<FormButton onClick={ dismissModal } isPrimary={ false }>
+			{ translate( 'Cancel' ) }
+		</FormButton>,
+	];
+
+	if ( 'add' !== mode ) {
+		buttons.unshift( {
+			action: 'delete',
+			label: <span><Gridicon icon="trash" /> { translate( 'Delete this package' ) }</span>,
+			onClick: onRemove,
+			additionalClassNames: 'packages__delete is-scary is-borderless'
+		} );
+	}
+
+	return buttons;
+};
+
+const OuterDimensionsToggle = ( { siteId, toggleOuterDimensions, translate } ) => {
+	const onClick = ( evt ) => {
```

Comment:
Sometimes e, sometimes evt. How about event in all the places (I _think_ we tend to use event.)